hr_expense_invoice/models/account_move.py

```diff
@@ -19,8 +19,11 @@ class AccountMove(models.Model):
         "transfer journal entry",
     )
 
-    @api.constrains("amount_total")
-    def _check_expense_ids(self):
+    def write(self, vals):
+        # Check if the amount of the invoice linked to an invoice is different
+        # Done here in the write instead of a Python constraint as the computed field
+        # amount_total is not yet updated on that moment
+        res = super().write(vals)
```

Comment:
```suggestion
        res = super().write(vals)
        if self.env.context.get("skip_account_move_synchronization"):
            return res
```

Reply:
@pedrobaeza can you address it? 

@loida-vm  @edlopen please approve this PR in order to allow me to continue doing things

Reply:
Changed, @Shide